src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -1101,16 +1137,24 @@ void GenerateFieldDocComment(io::Printer* printer,
   printer->Print(
     " * <code>^def^</code>\n",
     "def", EscapePhpdoc(FirstLineOf(field->DebugString())));
+  if (function_type == kFieldSetter) {
+    printer->Print(" * @param ^php_type^ $var\n",
+      "php_type", PhpTypeName(field, is_descriptor));
```

Comment:
Is @return for fluent setter necessary? I feel it's quite redundent. But I'm open to add it if we need to be strict with the doc format.